### ISSUE TITLE
alert, listener and trace logs are not available options for enabled_cloudwatch_logs_exports when oracle engine is used in aws_db_instance

### ISSUE THREAD

**Issue by ruortega** [terraform-providers/terraform-provider-aws#5235]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

```hcl
Terraform v0.11.7
provider.aws v1.27.0
```

### Affected Resource(s)

```hcl
resource "aws_db_instance" "database" {
  enabled_cloudwatch_logs_exports = ["alert","audit","listener","trace"]
}
```

### Terraform Configuration Files

```hcl
resource "aws_db_instance" "database" {
  engine                          = "oracle-ee"
  enabled_cloudwatch_logs_exports = ["alert","audit","listener","trace"]
}
```

### Expected Behavior

We expect terraform to enable alert, audit, listener and trace cloudwatch logs exports when oracle is selected as db engine

### Actual Behavior

```hcl
Error: aws_db_instance.database: expected enabled_cloudwatch_logs_exports.0 to be one of [audit error general slowquery], got alert

Error: aws_db_instance.database: expected enabled_cloudwatch_logs_exports.2 to be one of [audit error general slowquery], got listener

Error: aws_db_instance.database: expected enabled_cloudwatch_logs_exports.3 to be one of [audit error general slowquery], got trace
```

### Steps to Reproduce

1. `terraform apply`

### References

* https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_LogAccess.Concepts.Oracle.html

Thank you

**Comment by bflad** [comment]
Pull request submitted: #5494

**Comment by bflad** [comment]
In version 1.32.0 of the AWS provider, releasing later this week, you will be able to declare these additional log types. 👍

**Comment by bflad** [comment]
This has been released in [version 1.32.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1320-august-16-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
